devel/nvm/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nvm-sh nvm 0.35.2 v
+
+categories          devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+maintainers         @FranklinYu openmaintainer
```

Comment:
This should also include an obscured version of your email address; see other ports for examples.